src/scss/components/topnav/mixins/_topNavControl.scss

```diff
@@ -9,15 +9,20 @@
 
```

Comment:
Do you know where the above property is being used?

```scss
box-shadow: $focus-glow;
```

Reply:
i have removed pristine state completely . We are not using these styles anywhere currently